internal/plugins/helm/v1/chartutil/testdata/test-chart/templates/_helpers.tpl

```diff
@@ -58,6 +58,6 @@ Create the name of the service account to use
 {{- if .Values.serviceAccount.create -}}
     {{ default (include "test-chart.fullname" .) .Values.serviceAccount.name }}
 {{- else -}}
```

Comment:
The new service account feature has the help plugin always creating a service account, so this condition should be removed
```suggestion
```

Reply:
We need to remove the default one.